src/components/YesNoContest.tsx

```diff
@@ -252,10 +257,7 @@ export default class YesNoContest extends React.Component<Props> {
   }
 
   public closeOvervoteAlert = () => {
-    // Delay to avoid passing tap to next screen
-    window.setTimeout(() => {
-      this.setState({ overvoteSelection: initialState.overvoteSelection })
-    }, 200)
+    this.setState({ overvoteSelection: initialState.overvoteSelection })
```

Comment:
I wouldn't want to remove the timeout right now unless we've done extensive testing. I know it *shouldn't* break, but this is super tricky.

Reply:
Fair enough, though we did remove the other ones:
- e302f200b40ce1fa1977eb45ee939a1fdb8151ad
- 841cebe559c4bb80c9345377a76453ed405add44